### ISSUE TITLE
Using grepUntagged to notify on untagged tests

### ISSUE THREAD

**Issue by yonatanaftali** [cypress-io/cypress-grep#101]
Hey,

I've split all of my tests into 4 groups (`@A, @B, @C, @D`), and created a GitHub Action that runs 5 instances of Cypress in parallel. Each instance covers one group, and the 5th instance runs using `grepUntagged=true`.
I'm also using the flags `grepFilterSpecs` and `grepOmitFiltered`, to completely ignore tagged tests from other groups.

When running the 5th instance, all of the specs are shown in the logs, but they are completed after 1-2 ms (as expected).
I wanted to ask if there's a way I can create some kind of notification if the 5th instance (running untagged tests) actually runs a test? Ideally, I want all of my tests to be covered, so I want to comment on the PR if the CI found an untagged test.

I'd appreciate your help!

UPDATE: I managed to create the notification I needed using a shell script that queries the test amount from all of the mochawesome JSONs, but it feels a bit too complicated. So now I just want to know if cypress-grep can provide this notification more cleanly?

**Comment by bahmutov** [comment]
I am not sure what notification you need to be honest. What do you mean by "actually runs a test"? Something like https://github.com/bahmutov/cypress-expect does?

**Comment by yonatanaftali** [comment]
The solution I implemented provides the following notification for example:
![image](https://user-images.githubusercontent.com/57040227/147949032-7cf19e1c-027a-4136-a081-79a9aa3cfbc0.png)

I tried using cypress-expect with the command: `npx cypress-expect run --passing 0 --failing 0  --pending 0 --env grepUntagged=true`, but the run immediately ends with the message 

> expected a number of --passing tests 0

**Comment by bahmutov** [comment]
Maybe there is min passing?

**Comment by yonatanaftali** [comment]
I don't see how --min-passing can help me here, but in any case, I tried running `npx cypress-expect run --min-passing 0 --env grepUntagged=true` and the run still ended immediately.

I don't really care if the untagged tests pass or not - I just want to know if they were executed.

**Comment by bahmutov** [comment]
I still don’t get the use case a little bit. Probably a repo with minimal reproduction would help

**Comment by yonatanaftali** [comment]
The use case is this: I want all of my tests to be tagged so that I can control and balance the different groups when running them in parallel. If someone on my team added a new test and didn't tag it - I'd like to have a notification about that, so the test will be tagged. An example for such notification is the following:
![image](https://user-images.githubusercontent.com/57040227/147955387-92322cbf-cf60-4485-8288-781b3b130dc7.png)

I managed to get this notification using a shell script I wrote, but I'd prefer if cypress-grep or cypress-expect will offer this functionality:

- For cypress-expected - If I could do n`px cypress-expect run --executed 0 --env grepUntagged=true`.
- For cypress-grep - If when running using grepUntagged=true I'll get the number of executed tests.

**Comment by jordanpowell88** [comment]
We are in the process of deprecating this repo as this package has been republished under `@cypress/grep` and has moved to [https://github.com/cypress-io/cypress/tree/develop/npm/grep](https://github.com/cypress-io/cypress/tree/develop/npm/grep). I'm going to go ahead and close this issue as it appears stale. If this is still an issue, please open a new issue [here](https://github.com/cypress-io/cypress/issues/new/choose)
